lib/sidekiq/throttled.rb

```diff
@@ -93,6 +91,25 @@ def throttled?(message)
 
       private
 
+      # Preset the the sidekiq fetch strategy
+      #
+      # With sidekiq -> 6.1 the internal API of sidekiq changed slightly.
+      # Versions below sidekiq 6.1 needs to initialize the fetcher
+      # instead of passing the fetch class.
+      #
+      # @return [void]
+      def preset_strategy!
+        require "sidekiq/throttled/fetch"
+
+        Sidekiq.options[:fetch] = if Gem::Version.new(Sidekiq::VERSION) < Gem::Version.new("6.1.0")
+                                    Sidekiq::Throttled::Fetch
+                                  else
+                                    Sidekiq::Throttled::Fetch.new(
+                                      Sidekiq.options
+                                    )
+                                  end
+      end
```

Comment:
Please rename the method to `#setup_strategy!`, use indented style for conditionals, and Reduce comment to just a link to the changeset that changed API :D

```suggestion
      # @return [void]
      def setup_strategy!
        require "sidekiq/throttled/fetch"

        # https://github.com/mperham/sidekiq/commit/fce05c9d4b4c0411c982078a4cf3a63f20f739bc
        Sidekiq.options[:fetch] =
          if Gem::Version.new(Sidekiq::VERSION) < Gem::Version.new("6.1.0")
            Sidekiq::Throttled::Fetch
          else
            Sidekiq::Throttled::Fetch.new(Sidekiq.options)
          end
      end
```